data/json/vehicles/boats.json

```diff
@@ -174,5 +174,162 @@
       { "x": 3, "y": 0, "parts": [ "frame_wood_nw", "boat_board" ] },
       { "x": 3, "y": 1, "parts": [ "frame_wood_ne", "boat_board" ] }
     ]
+  },
+  {
+    "id": "cabin_cruiser_wood",
+    "type": "vehicle",
+    "name": "Wooden Cabin Cruiser",
```

Comment:
I'm getting an error when I load a new game instance and try to debug spawn this boat.  I suspect it's because of how vehicles are drawn (their order) and having the 0,0 placement in the back right corner of the boat.  Vehicles are "built" tile by tile from their 0.0 point when spawned.  The 0,0 point is also what you use for your x,y reference in map spawns.  

![image](https://user-images.githubusercontent.com/39442864/126051408-0625a977-96ad-4aa9-b5a6-83e538357e7a.png)

Reply:
To fix this I changed the 0, 0 point for the boat that I'll commit with a few other changes shortly. The error seems to be caused from the vehicle spawning with the player in a tile that's normally impassable.